### PR TITLE
cron pipeline groovy scripts

### PR DESCRIPTION
**Pull request by tintumathew10**
Signed-off-by: tintumathew10 <tmathew@redhat.com>

cron pipeline groovy scripts.

logs: 

https://ceph-downstream-jenkins-csb-storage.apps.ocp4.prod.psi.redhat.com/view/RHCS%20QE/job/rhceph-cron-test-executor/10/console
https://ceph-downstream-jenkins-csb-storage.apps.ocp4.prod.psi.redhat.com/view/RHCS%20QE/job/rhceph-cron-pipeline/3/console

gchat notification:
Downstream Ceph Jenkins, Bot, 1:29 PM
Run for RHCEPH-5.1 Cron Pipeline Stage-1 is STABLE.Log:https://ceph-downstream-jenkins-csb-storage.apps.ocp4.prod.psi.redhat.com/job/rhceph-cron-test-executor/10/

https://datagrepper.engineering.redhat.com/id?id=ID:ceph-downstream-jenkins-csb-storage-35909-1644060251343-43943:1:1:1:1&is_raw=true&size=extra-large

### CODE REVIEW

File: pipeline/cron-pipeline-test-executor.groovy [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+node(nodeName) {
+    try {
+        timeout(unit: "MINUTES", time: 30) {
+            stage('Install prereq') {
+                if (env.WORKSPACE) { sh script: "sudo rm -rf * .venv" }
+                checkout(
+                    scm: [
+                        $class: 'GitSCM',
+                        branches: [[name: 'refs/remotes/origin/cron_grv']],
```
Comment (psathyan) [inline pipeline/cron-pipeline-test-executor.groovy:23]: Could we change this to `origin/master`?

File: pipeline/cron-pipeline.groovy [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+node(nodeName) {
+    try {
+        timeout(unit: "MINUTES", time: 30) {
+            stage('Install prereq') {
+                if (env.WORKSPACE) { sh script: "sudo rm -rf * .venv" }
+                checkout(
+                    scm: [
+                        $class: 'GitSCM',
+                        branches: [[name: 'refs/remotes/origin/cron_grv']],
```
Comment (psathyan) [inline pipeline/cron-pipeline.groovy:19]: Could we change the branch to `origin/master`

File: pipeline/scripts/cron/5/1/stage-1/test-rados-scrub-scenarios.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
+#! /bin/sh
```
Comment (psathyan) [inline pipeline/scripts/cron/5/1/stage-1/test-rados-scrub-scenarios.sh:1]: Can we have this also included in stage-0 as we have three slots available?

**Review by psathyan — COMMENTED** [review]
Thanks for putting this together quickly. Some minor changes.

Would you be removing the suites from Nightly pipeline in this PR or a separate one?

File: pipeline/cron-pipeline-test-executor.groovy (hunk 1 above)
Comment (tintumathew10) [reply pipeline/cron-pipeline-test-executor.groovy]: Thanks for pointing it. made the changes.

File: pipeline/cron-pipeline.groovy (hunk 2 above)
Comment (tintumathew10) [reply pipeline/cron-pipeline.groovy]: done

**Review by psathyan — APPROVED** [review]

File: pipeline/scripts/cron/5/1/stage-1/test-rados-scrub-scenarios.sh (hunk 3 above)
Comment (tintumathew10) [reply pipeline/scripts/cron/5/1/stage-1/test-rados-scrub-scenarios.sh]: added to stage-1.

**Pull request merged**